pyserini/2cr/msmarco-v1-passage.yaml

```diff
@@ -541,3 +541,26 @@ conditions:
           - MAP: 0.4754
             nDCG@10: 0.6882
             R@1K: 0.8429
+  - name: slimr
+    display: "SLIM: on-the-fly query inference"
+    display-html: "SLIM: on-the-fly query inference"
+    display-row: "[6]"
+    command: python -m pyserini.search.lucene --index msmarco-v1-passage-slimr --topics $topics --encoder castorini/slimr-msmarco-passage --encoded-corpus scipy-sparse-vectors.msmarco-v1-passage-slimr --output $output --output-format msmarco  --batch 128 --threads 16 --hits 1000 --impact --min-idf 3
```

Comment:
Can we move `--batch 128 --threads 16` to the front to be consistent with the other commands?